components/datetime/src/zoned_datetime.rs

```diff
@@ -210,26 +203,36 @@ impl<C: CldrCalendar> TypedZonedDateTimeFormatter<C> {
     /// Constructor that takes a selected locale and a list of [`DateTimeFormatterOptions`].
     /// It collects all data necessary to format zoned datetime values into the given locale.
     ///
-    /// ✨ *Enabled with the `compiled_data` Cargo feature.*
+    /// ✨ *Enabled with the `compiled_data` and `experimental` Cargo features.*
```

Comment:
this looks messy because I change the order, putting the non-experimental constructors first. this line is the only other change